packages/terra-spacer/src/Spacer.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { withBps } from 'react-bps';
+import 'terra-base/lib/baseStyles';
+import styles from './Spacer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes.
+   */
+  children: PropTypes.node,
+  /**
+   * Sets top margin. One of 'none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4'.
+   */
+  marginTop: PropTypes.oneOf(['none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4']),
+  /**
+   * Sets bottom margin. One of 'none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4'.
+   */
+  marginBottom: PropTypes.oneOf(['none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4']),
+  /**
+   * Sets left margin One of 'none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4'.
```

Comment:
What was the decision on small-2 to large+4? Why not `small-3` to `large+3`? I know we use the breakpoint values that range from tiny to huge.

Also, what does -1 or +1 indicate? Plus 1 px value? plus .1 em value?  We should specify what these scales mean in the README.

Reply:
@emilyrohrbough I followed the pre-reviewed tech design https://github.com/cerner/terra-core/issues/696.